tests/infra/network.py

```diff
@@ -990,7 +996,7 @@ def _get_ledger_public_view_at(self, node, call, seqno, timeout):
                 return call(seqno)
             except Exception:
                 self.consortium.create_and_withdraw_large_proposal(node)
-
+                time.sleep(0.1)
```

Comment:
What's this sleep doing?

Reply:
Not much. I was just worried of doing an action in a loop without waiting at least a little bit between each iteration.